test/pyca/cryptography.sh

```diff
@@ -0,0 +1,67 @@
+#!/bin/sh
```

Comment:
There's a fairly recent new feature in the test framework.  `perldoc test/testlib/OpenSSL/Test.pm`, the DESCRIPTION section says some about it, and there's the `data_file` function that can be used to construct a path, for example:

``` perl
    ok(run(cmd(["sh", data_file("cryptography.sh")])),
        "running Python Cryptography tests");
```

This has the benefit to keep things together, so what I would like to see is this file moved to `test/recipes/90-test_external_data/` and the call of it adapted accordingly in `90-test_external.t`